dffml/operation/datetime.py

```diff
@@ -0,0 +1,39 @@
+import datetime
+from ..df.base import op
+from ..df.types import Definition
+
+current_datetime = Definition(name="current_datetime", primitive="generic")
+date_time = Definition(name="date_time", primitive="generic")
+
+
+@op(name="dffml.datetime", outputs=current_datetime)
+def date_time():
+    """
+    A function to generate current date and time.
+
+    Parameters
+    ----------
+    output:
+        Current date and time.
+    """
+    current_datetime = datetime.datetime.now()
+    return current_datetime
+
+
+@op(name="dffml.datetimeformat", outputs=date_time)
+def date_time_format():
+    """
+    A function to modify date and time format.
+
```

Comment:
Example here, pass in staticlly created `datetime.datetime` so that date is always the same